resources/META-INF/plugin.xml

```diff
@@ -39,15 +40,17 @@
   <depends>com.intellij.modules.lang</depends>
 
   <actions>
-    <action id="ClangFormat.ClangFormat" class="io.probst.idea.clangformat.ClangFormatAction"
-            text="clang-format"
-            description="Runs clang-format on this buffer">
-      <add-to-group group-id="EditorPopupMenu" anchor="last"/>
-
-      <keyboard-shortcut keymap="Mac OS X" first-keystroke="control alt K"/>
-      <keyboard-shortcut keymap="Mac OS X" first-keystroke="meta alt K"/>
-      <keyboard-shortcut keymap="Mac OS X 10.5+" first-keystroke="meta alt K"/>
-    </action>
+    <group id="ClangFormat.Group" text="ClangFormat" popup="true">
```

Comment:
Do we need a group here? It seems that we only have one action in it, wouldn't this cause a somewhat odd `ClangFormat -> Format this file` menu structure? Just one level would be easier, wouldn't it?